### DISCUSSION TITLE
I  changed  the  installing path  and then encounted    one   error

### DISCUSSION THREAD

**Discussion by zhang377** [nextpyp/.github#125] [Q&A] [answered]
Hi ,professor,I changed  the installing path  dut  to  there  is  no  enough  space  in "/".
My operating  system is centos 7.
I operated command step by  step  as tutorial  shows,  until   create a new fold to install nextPYP.
The  new path is "/data05/nextPYP"  instead  of "/opt/nextpyp".And  in  this  new  folder（This path belongs to the hard drive that is mounted to the system later）,I downloaded the  `install` script and excuted it.
All files that `install` script contains were downloaded successfully.Unfortunately,there  was  an error :
![image](https://github.com/user-attachments/assets/c09fa396-4645-4e95-83bb-40392a7c5320)
I attached  the logs  of  the  "journalctl -xe"  command  which  name is `journalctl.txt`.
[journalctl.txt](https://github.com/user-attachments/files/17364423/journalctl.txt)

**Comment by abartesaghi** [comment, marked as answer]
The only error I see is related to singularity not being able create a user namespace, which appears to be disabled in your system. An administrator should be able to re-enable namespaces which are needed by nextPYP. According to some reports online and depending on your system configuration, updating to a newer version of singularity/apptainer may also solve the problem.

**Reply by zhang377** [reply]
Thank  you,professor!
I  will try to  update the  version  of  singularity/appatainer.
